keyboards/mykeyclub/jris65/hotswap/keyboard.json

```diff
@@ -0,0 +1,104 @@
+{
+    "manufacturer": "zlkb",
+    "keyboard_name": "IRIS65",
+    "maintainer": "5217zl",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": false
```

Comment:
```suggestion
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
```

Reply:
Are the removed settings because they are the default or is there another reason?

Reply:
specifying `false` for a feature that is false by default seems needless 😊